### ISSUE TITLE
how to deal

### ISSUE THREAD

**Issue by wangduyang** [TexasInstruments/edgeai-yolov5#17]
![图片](https://user-images.githubusercontent.com/26652416/172318525-3d426d14-22eb-402d-b7df-6a473efbdce5.png)

**Comment by debapriyamaji** [comment]
Please use the exact command for training:
python train.py --data coco_kpts.yaml --cfg yolov5s6_kpts.yaml --weights 'path to the pre-trained ckpts' --batch-size 64 --img 640 --kpt-label

I think you are missing the flag "kpt-label".

Regards, Debapriya

**Comment by wangduyang** [comment]
@debapriyamaji thank you

**Comment by wangduyang** [comment]
I have to bother you again,I want to use your method to train my dataset,But it has more than 17 points，does your method support it？

**Comment by YHDang** [comment]
> @debapriyamaji thank you I did ignore the “kpt-label”

Hello, I also met the same question as you. And I tried to add '--kpt-label' to the running command, but why is the unrecognized argument?
![image](https://user-images.githubusercontent.com/34653678/190426783-54a685b2-ebab-4f4e-bb0d-d91943b17938.png)
